src/lang/en-US.ts

```diff
@@ -184,17 +182,34 @@ const LoginPage: ResolvedIntlConfig['messages'] = {
     'login.oidc.message': `Please use one of the providers below to continue.`,
 };
 
+const Captures: ResolvedIntlConfig['messages'] = {
+    'capturesTable.title': `Your Captures`,
+    'capturesTable.detailsCTA': `Details`,
+    'captures.main.message1': `Click the "New Capture" button up above to get started.`,
+    'captures.main.message2': `It will guide you through generating and downloading a valid {docLink}.`,
+    'captures.main.message2.docLink': `capture spec`,
+    'captures.main.message2.docPath': `https://docs.estuary.dev/concepts/#captures`,
+};
+
+const Materializations: ResolvedIntlConfig['messages'] = {
+    'materializationsTable.title': `Materializations`,
+    'materializationsTable.detailsCTA': `Details`,
+    'materializations.main.message1': `Click the "New Materialization" button up above to get started.`,
+    'materializations.main.message2': `It will guide you through generating and downloading a valid {docLink}.`,
+    'materializations.main.message2.docLink': `materialization`,
```

Comment:
Should the `docLink` text mirror that for captures (i.e., _materialization spec_) or vice versa?